### PR TITLE
Added new option and removed old ajax options

### PR DESCRIPTION
**Pull request by fizerkhan**
We added new features to our product. Hence we removed old options `enableAjaxAbort` and `enableAjaxErrors`. We added our new option `disableAjaxMonitoring`. 

Can you please merge it?

Also please remove `enableAjaxAbort` and `enableAjaxErrors` options from segment atatus integration page.

And add an option `disableAjaxMonitoring` with following  name in the integration page.

```
  Disable tracking of AJAX requests.
```

The description of this option: 

```
  If you don't want to monitor AJAX requests, enable this setting.
```

### CODE REVIEW

File: History.md [hunk 1]
```diff
@@ -1,4 +1,10 @@
 
+1.0.10 / 2015-09-25
```
Comment (f2prateek) [inline History.md]: Remove this, we'll do do this as a part of the release process.

File: component.json [hunk 2]
```diff
@@ -2,7 +2,7 @@
   "name": "analytics.js-integration-atatus",
   "repo": "segment-integrations/analytics.js-integration-atatus",
   "description": "The Atatus analytics.js integration.",
-  "version": "1.0.9",
+  "version": "1.0.10",
```
Comment (f2prateek) [inline component.json]: Remove this, we'll do do this as a part of the release process.

**Comment by fizerkhan** [comment]
@f2prateek Removed lines from History and component. Can you check and merge it?

**Comment by f2prateek** [comment]
@fizerkhan lgtm, can you please squash and rebase your changes and I can merge it.

**Comment by fizerkhan** [comment]
@f2prateek  Since i already committed and pushed the code, i could not squash and rebase it. 
I tried it, but it creates lot of conflicts. So i abort the rebase.

**Comment by fizerkhan** [comment]
@f2prateek any updates on this?

**Pull request closed without merging**